packages/prisma/schema.prisma

```diff
@@ -559,18 +560,19 @@ enum WorkflowActions {
 }
 
 model WorkflowStep {
-  id                Int                @id @default(autoincrement())
-  stepNumber        Int
-  action            WorkflowActions
-  workflowId        Int
-  workflow          Workflow           @relation(fields: [workflowId], references: [id], onDelete: Cascade)
-  sendTo            String?
-  reminderBody      String?
-  emailSubject      String?
-  template          WorkflowTemplates  @default(REMINDER)
-  workflowReminders WorkflowReminder[]
-  numberRequired    Boolean?
-  sender            String?
+  id                        Int                @id @default(autoincrement())
+  stepNumber                Int
+  action                    WorkflowActions
+  workflowId                Int
+  workflow                  Workflow           @relation(fields: [workflowId], references: [id], onDelete: Cascade)
+  sendTo                    String?
+  reminderBody              String?
+  emailSubject              String?
+  template                  WorkflowTemplates  @default(REMINDER)
+  workflowReminders         WorkflowReminder[]
+  numberRequired            Boolean?
+  sender                    String?
+  numberVerificationPending Boolean            @default(true)
```

Comment:
All existing workflow steps will have this set to true and for all new created or updated steps this will be false. This is needed, as otherwise, all already existing workflows with 'send SMS specific number' won't work anymore